### PR TITLE
Closes #8: Edit on double-click.

### LINKED ISSUE #8: Edit on double-click
That would be useful to pass in edit-mode when double clicking on an item.

### PR DESCRIPTION
**Pull request by n1k0**
refs #8, r=? @almet

### CODE REVIEW

File: scripts/components/CollectionList.js [hunk 1]
```diff
@@ -40,6 +40,12 @@ class Row extends Component {
     return new Date(lastModified).toJSON();
   }
 
+  onDoubleClick(event) {
+    event.preventDefault();
+    const { name, record} = this.props;
```
Comment (almet) [inline scripts/components/CollectionList.js]: nit: missing a space.

File: scripts/containers/CollectionListPage.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { connect } from "react-redux";
 import CollectionList from "../components/CollectionList";
 import * as CollectionActions from "../actions/collection";
 import * as NotificationsActions from "../actions/notifications";
+import { updatePath } from "../redux-router";
```
Comment (almet) [inline scripts/containers/CollectionListPage.js]: Can't you directly get a mixin to receive the ability to change the routing? (I believe it was possible in react-router for instance).

**Comment by almet** [comment]
Looks good, with nits.

File: scripts/containers/CollectionListPage.js (hunk 2 above)
Comment (n1k0) [inline scripts/containers/CollectionListPage.js]: I can, but I'm using redux-router to map urls to state, and updating the path is done that way.

**Pull request merged**